src/main/java/edu/harvard/iq/dataverse/export/SchemaDotOrgExporter.java

```diff
@@ -92,6 +102,266 @@ public void exportDataset(DatasetVersion version, JsonObject json, OutputStream
         }
     }
 
+
+    // The export subsystem assumes there is only
+    // one metadata export in a given format per dataset (it uses the current
+    // released (published) version. This JSON fragment is generated for a
+    // specific released version - and we can have multiple released versions.
+    // So something will need to be modified to accommodate this. -- L.A.
+    /**
+     * We call the export format "Schema.org JSON-LD" and extensive Javadoc can
+     * be found in {@link SchemaDotOrgExporter}.
+     */
+    public String getJsonLd(DatasetVersion version) {
+        String jsonLd;
+
+        JsonObjectBuilder job = Json.createObjectBuilder();
+        job.add("@context", "http://schema.org");
+        job.add("@type", "Dataset");
+        // Note that whenever you use "@id" you should also use "identifier" and vice versa.
+        job.add("@id", version.getDataset().getPersistentURL());
+        job.add("identifier", version.getDataset().getPersistentURL());
```

Comment:
I'm guessing that the only changes in this block are changing this to version. If not, could you highlight any other changes you made?